rfc/rfc010-multiple-operators.md

```diff
@@ -0,0 +1,64 @@
+# RFC010 Multiple operator service access
```

Comment:
I don't like the term 'piece of software' because it sounds like 'spreektaal'. Maybe 'software system' or just 'system'?

Reply:
rewrote everywhere (1 instance :p)